OpenKh.Tools.Kh2TextEditor/Views/TextEditorView.xaml

```diff
@@ -71,9 +77,88 @@
             </Grid.RowDefinitions>
 
             <TextBox
+                x:Name="textInputField"
                 Grid.Row="0"
                 MaxLines="7" TextWrapping="Wrap" VerticalScrollBarVisibility="Auto"
-                Text="{Binding Text, UpdateSourceTrigger=PropertyChanged}"/>
+                Text="{Binding Text, UpdateSourceTrigger=PropertyChanged}">
+                <TextBox.ContextMenu>
+                    <ContextMenu>
+                        <MenuItem Header="Icons">
```

Comment:
The list is really looooong. Can I suggest to split it into categories? Like Icons\Buttons, Materials, Ranks, Weapons, etc. It would allow to reach different elements from the list faster.

Reply:
Could do.